workers/generic-worker/main.go

```diff
@@ -6,7 +6,7 @@
 //go:generate go run ./gw-codegen file://schemas/multiuser_posix.yml    generated_multiuser_darwin.go    multiuser
 //go:generate go run ./gw-codegen file://schemas/multiuser_posix.yml    generated_multiuser_linux.go     multiuser
 //go:generate go run ./gw-codegen file://schemas/multiuser_windows.yml  generated_multiuser_windows.go   multiuser
-// //go:generate gw-codegen https://raw.githubusercontent.com/taskcluster/docker-worker/66dfa0ec97602285fa5f05c2d8cbf487f52c7e27/schemas/payload.json dockerworker/payload.go
+// //go:generate gw-codegen https://raw.githubusercontent.com/taskcluster/taskcluster/4c9c284a8afda18ec3e163a8be198450128a3ef0/workers/docker-worker/schemas/v1/payload.yml dockerworker/payload.go
```

Comment:
I think this is not doing anything as it is double commented out. Correct me if I'm wrong :)

Reply:
I was curious if you had tried uncommenting it to check if the updated version works. It is true it isn't used at the moment.

If I remember correctly, this was an initial stab at generating go types to represent the docker-worker payload, to aid the implementation of the docker-worker-compatible payload in generic-worker. I think it was probably commented out when that work was put on hold.

Reply:
Hmm. No I didn't before, now when I do, it fails in both places - with old url and new one.
```
gw-codegen: Failed to determine go package inside directory 'taskcluster/workers/generic-worker/dockerworker' - is your GOPATH set correctly ('~/.gvm/pkgsets/go1.18.1/global')? Error: cannot find package "." in:
	taskcluster/workers/generic-worker/dockerworker
```

Reply:
Ah, I think the `dockerworker` subdirectory needs to exist. You might also need to put an empty go file inside it just including the package name (`package dockerworker`).

It probably isn't ideal us having half-implemented code commented out in our codebase. In one sense, it can be positive in that it is a bit of a carrot dangling in front of us to incentivise us to implement it, but at the same time it is also a bit of noise. I hope we'll get to the docker-worker payload work later this year, so maybe it can just be a short lived thing.

I also don't love the idea of a directory called `dockerworker` inside `generic-worker`. The package organisation is a bit of a mess in general - there is a load of code in the `main` package, which isn't ideal. It might be nice to separate it out a bit at some point too. :-/